check-solution/action.yml

```diff
@@ -48,6 +48,11 @@ inputs:
     description: The name of the artifact folder of which the PowerApps Checker logs will be uploaded.  Default value is \'CheckSolutionLogs\'
     required: false
 
+  fail-on-analysis-error:
```

Comment:
Hmm.   In general, we don't have optional boolean arguments which default to true.
I agree that defaulting to true here should be the expected behavior, but perhaps we should do the negative instead?  I.E., name the flag `continue-on-analysis-error` with a default `'false'` ?